### PR TITLE
Mengting login button

### PR DESCRIPTION
**Pull request by MengtingCao**
I added gems that google oauth implementation will require. As well as in nav bar added 'MCHSO' next to the logo.

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '2b18bbd5627f961708ce71b5d181c2522ad3ade569e3ee11141f0644b49b763a80dc11d6aab5c16fd6e8beb895687f32c411dcdb092008842691c2cc6a4ddb10'
```
Comment (jswood23) [inline config/initializers/devise.rb:17]: We should put this data in a separate file called `.env` that doesn't go into GitHub and import it into this file.

Comment (jswood23) [reply config/initializers/devise.rb:17]: If you'd prefer, we could wait to make this change until a later PR since there's no actual secret key yet.

Comment (MengtingCao) [reply config/initializers/devise.rb:17]: I believe that's just a place where you can provide devise a secret key. Devise by default uses its own key.

**Review by jswood23 — APPROVED** [review]
Looks good

**Review by ynzhang09 — APPROVED** [review]

**Pull request merged**